### DISCUSSION TITLE
Additional fields for the IATI transaction data

### DISCUSSION THREAD

**Discussion by dean-breed** [devinit/ddw-analyst-ui#441] [Ideas]
For other workstreams, we will need to include:

- funding organisation; and
- both coordinate types.

How will we go about fitting these in?

**Comment by bill-anderson** [comment]
Funding organisation is already there non-normalised in the Participating Organisation columns. (Role = 1). As an activity can have any number of participating organisations it will be difficult to flatten this. 

If the transaction provider org is populated this will provide the required data, but it is not mandatory.

Note also that some bilateral agencies will make a distinction between the funding organisation (the government) and the extending organisation (the agency) as in the CRS donor and donor agency split

**Comment by bill-anderson** [comment]
Coordinates: we need, at some point, to create a separate table to represent sub-national flows. This still needs conceptualising.

**Comment by k8hughes** [comment]
I am not clear who should do the work for this, if conceptualising is required, who should look into it?

**Comment by dean-breed** [comment]
Just to add context, I asked these questions as they are both aspects that people are keen to look at in ITEP workstreams.

**Comment by bill-anderson** [comment]
A work round on sub-national locations is to use d-portal's location csv output and join it to our transaction table (iati-identifier is in the u.r.l column). It is also possible to modify d-portal's query: see all available fields in this code - https://github.com/devinit/D-Portal/blob/master/dstore/js/dstore_db.js

**Comment by k8hughes** [comment]
@dean-breed  and @bill-anderson I'm going to convert this to a discussion till we decide what work might be required.

**Comment by k8hughes** [comment]
We are talking about partitioning IATI data by date, so we have a table by year.
Then having a separate activity table where single value activity data can live.
Policy markers, budget data are also bits of data that we want to bring in.

**Comment by k8hughes** [comment]
Could also partition by sector vocab = 1 - as most of our work is done on this sector.

**Reply by bill-anderson** [reply]
NB this is sector vocab no = 1
